tests/cbmc/stubs/getpeername.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <assert.h>
+#include <cbmc_proof/nondet.h>
+#include <sys/socket.h>
+#include <errno.h>
+
+/* https://pubs.opengroup.org/onlinepubs/009695399/functions/getpeername.html */
+int getpeername(int socket, struct sockaddr *address, socklen_t *address_len)
+{
+    // assert(socket >= -1 && socket <= 65536); /* File descriptor limit. */   
+    errno = nondet_int();
```

Comment:
`errno` is set to `nondet_int()` even if return code is `0`.

Is that the right way to stub this function @feliperodri?
Shouldn't `errno` ideally remain unchanged if we have no new errors?

(Same comment on other stubs)

Reply:
I agree-- I was going off of the stub cited in https://github.com/aws/s2n-tls/pull/2896#discussion_r650118718 but can definitely change this easily.

Reply:
We should only set `errno` if the return value is different than zero.